alpaca_trade_api/entity_v2.py

```diff
@@ -87,6 +87,19 @@
     "t": "timestamp",
 }
 
+news_mapping_v2 = {
```

Comment:
Isn't this kind of pointless? Since the mapping doesn't actually change anything, I think we could just return the original dict.

Reply:
Removed